pdns/qtype.hh

```diff
@@ -79,7 +79,6 @@ public:
     MB=7,
     MG=8,
     MR=9,
-    WKS=11,
```

Comment:
There is no boilerplate item for WKS, and this will prevent the fallback to TYPE11
Since the use of WKS is discouraged in RFC 1123 and RFC 1127 I don't think it is worth the trouble of adding the type.